core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -108,6 +108,8 @@ def _set_default_params(self):
             "stratify": "auto",
             "bin": "auto",
             "n_bins": None,
+            "vary_seed_across_folds": True, # If True, the seed used for each fold will be varied across folds.
```

Comment:
For this PR, lets set it to False so we enable the ability to use the logic, but make it so the logic doesn't yet change AutoGluon's default behavior until we benchmark things and decide if it is beneficial.